cypress/integration/rollups_spec.js

```diff
@@ -18,9 +18,15 @@ describe("Rollups", () => {
 
     cy.wait(10000)
 
-    // Click on "Sample data" tab
+    // TODO determine why line 25 and/or line 28 are needed occasionally for rollups_spec to pass.  
```

Comment:
This seems like an interesting situation. Not blocking, as it wouldn't require code changes, but I'd recommend creating a github issue to track this so it doesn't slip through the cracks.